spec.html

```diff
@@ -23352,6 +23351,12 @@ <h1>NaN</h1>
       <h1>undefined</h1>
       <p>The value of `undefined` is *undefined* (see <emu-xref href="#sec-ecmascript-language-types-undefined-type"></emu-xref>). This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
     </emu-clause>
+
+    <emu-clause id="sec-global-this">
+      <h1>globalThis</h1>
+      <p>The initial value of `globalThis` is the field of the current Realm record named [[GetGlobalThis]].
```

Comment:
```suggestion
      <p>The initial value of `globalThis` is the value of the [[GetGlobalThis]] field of the current Realm Record.
```
You also need to declare the field in Table 20, initialize it to `*undefined*` in `CreateRealm`, and set it in `SetRealmGlobalObject`. And I think you should drop the "Get".

Reply:
@jmdyck actually this was a typo; it's meant to be `realm.[[GlobalEnv]].[[GlobalThisValue]]`; will update.